firebase-datatransport/firebase-datatransport.gradle

```diff
@@ -14,8 +14,21 @@
 
 plugins {
     id 'firebase-library'
+    id 'com.google.protobuf'
```

Comment:
we can't afford a proto dependency here due to size and compatibility issues, pls use json instead, the easiest way to do it is to use `@Encodable`.